sky/authentication.py

```diff
@@ -404,16 +408,49 @@ def setup_kubernetes_authentication(config: Dict[str, Any]) -> Dict[str, Any]:
             logger.error(suffix)
             raise
 
+    ssh_jump_name = clouds.Kubernetes.SKY_SSH_JUMP_NAME
+    if ssh_setup_mode == 'nodeport':
```

Comment:
Can we use the enum `KubernetesNetworkingMode` everywhere to avoid hardcoding strings 'nodeport' and 'portforward'? We can also ask the user to use the same string in the config file and then read directly.

Reply:
@romilbhardwaj I'm wondering if we should create an `enum` for service types as well since there are currently some places(`setup_kubernetes_authentication`,  `setup_sshjump_svc`) using `NodePort` and `ClusterIP` as a hardcoded string. What do you think?

Reply:
Yes, I think it'll be good to have `KubernetesServiceType` enum